qemu/tests/cfg/block_with_iommu.cfg

```diff
@@ -0,0 +1,19 @@
+- block_with_iommu: install setup image_copy unattended_install.cdrom
+    only q35
+    only x86_64
+    only virtio_net
```

Comment:
Is this line necessary?

Reply:
This case only for linux and x86, so only for virtio_net nic model. It depends on manual case.

Reply:
It needs to confirm to featur owner(@XueqiangWei  @CongLi ).